File: VB/Controllers/HomeController.vb
Imports System
Imports System.Collections
Imports System.Collections.Generic
Imports System.Xml.Linq
Imports System.Web.Mvc
Imports DevExpress.Web.Mvc

Namespace CS.Controllers

    Public Class HomeController
        Inherits Controller

        Public Function Index() As ActionResult
            Dim model As MyModel = New MyModel() With {.Name = "DevExpress User", .Day = 5}
            ViewData("Items") = GetItems()
            Return View(model)
        End Function

        <HttpPost>
        Public Function Index(<ModelBinder(GetType(DevExpressEditorsBinder))> ByVal model As MyModel) As ActionResult
            TempData("PostedModel") = model
            Return RedirectToAction("Success")
        End Function

        Public Function Success() As ActionResult
            Return View(TempData("PostedModel"))
        End Function

        Private Function GetItems() As IList
            Dim items As List(Of TrackBarItemObject) = New List(Of TrackBarItemObject)()
            Dim document As XDocument = XDocument.Load(Server.MapPath("~/App_Data/TimeLapse.xml"))
            For Each item In document.Root.Elements("Day")
                items.Add(New TrackBarItemObject() With {.DayNum = Convert.ToInt32(item.Attribute("DayNum").Value), .ToolTip = CStr(item.Attribute("ToolTip").Value), .Text = CStr(item.Attribute("Text").Value)})
            Next

            Return items
        End Function
    End Class

    Public Class MyModel

        Public Property Name As String

        Public Property Day As Integer
    End Class

    Public Class TrackBarItemObject

        Public Property DayNum As Integer

        Public Property ToolTip As String

        Public Property Text As String
    End Class
End Namespace

File: CS/Controllers/HomeController.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Linq;
using System.Xml.Linq;
using System.Web;
using System.Web.Mvc;
using DevExpress.Web.Mvc;

namespace CS.Controllers {
    public class HomeController : Controller {
        public ActionResult Index() {
            MyModel model = new MyModel() {
                Name = "DevExpress User",
                Day = 5
            };

            ViewData["Items"] = GetItems();

            return View(model);
        }

        [HttpPost]
        public ActionResult Index([ModelBinder(typeof(DevExpressEditorsBinder))] MyModel model) {
            TempData["PostedModel"] = model;
            return RedirectToAction("Success");
        }

        public ActionResult Success() {
            return View(TempData["PostedModel"]);
        }

        private IList GetItems() {
            List<TrackBarItemObject> items = new List<TrackBarItemObject>();
            XDocument document = XDocument.Load(Server.MapPath("~/App_Data/TimeLapse.xml"));
            foreach(var item in document.Root.Elements("Day")) {
                items.Add(new TrackBarItemObject() {
                    DayNum = Convert.ToInt32(item.Attribute("DayNum").Value),
                    ToolTip = (string)item.Attribute("ToolTip").Value,
                    Text = (string)item.Attribute("Text").Value
                });
            }

            return items;
        }
    }
    
    public class MyModel {
        public string Name { get; set; }
        public int Day { get; set; }
    }

    public class TrackBarItemObject {
        public int DayNum { get; set; }
        public string ToolTip { get; set; }
        public string Text { get; set; }
    }
}
